# Why preprocess 

The reads we have recieved from the sequencing company or downloaded from the Sequence Nucleotide Archive or European Nucleotide Database are the **raw** results that have come out of the sequencing machine. 

We need to process these reads to do a few things. How we process the reads depends on what **type** of reads we have. 

## Illumina short reads:
  
-  CutAdapt: Remove any Illumina sequencing adapters from our reads 
-  CutAdapt: Trim low quality reads with CutAdapt, to remove possible sequencing errors. 
-  BBMerge: Merge our forward and reverse reads, to get one longer (and higher quality) read. 
-  FastQGroomer: Reformat the result from BBMerge to something we can use 
-  QC after processing: Rerun FastQC and MultiQC to get our quality after these processing steps. 

[Go to Illumina filtering instructions](CutAdapt.qmd)

## Oxford Nanopore long reads:

- Porechop: Remove any nanopore sequencing adapters from our reads
- fastp: Select only reads which are between 1000-2000 bp in length, and above a quality threshold

[Go to Nanopore filtering instructions](porechop.qmd)


